README.md

```diff
@@ -1,58 +1,49 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+Book Store
 
 > Description the project.
 
+        An App in which a user inputs a book title and Author and it is displayed.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Javscript
+- Frameworks - React
+- Technologies used - CSS
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
 ### Setup
 
+1. Have Vsc code installed.
+2. Node package
+
 ### Install
 
+1.Go to githubpage : https://github.com/Therealadetunji/React-Book-Store
+2.Click on code and Clone repository.
+and Open in Editor.
+
 ### Usage
 
 ### Run tests
 
 ### Deployment
 
-
-
 ## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.